quiz-app/README.md

```diff
@@ -1,35 +1,118 @@
 
-```sh
-npm install
-```
+After editing the translations, edit the index.js file in the translation folder to import all the files following the conventions in `en`.
 
-## Build the app
+Edit the `index.js` file in `assets/translations` to import the new translated files.
 
-To build the solution, run `npm run build`.
+Then, edit the dropdown in `App.vue` in this app to add your language. Match the localized abbreviation to the folder name for your language.
 
-```sh
-npm run build
-```
+Finally, edit all the quiz links in the translated lessons, if they exist, to include this localization as a query parameter: `?loc=fr` for example.
 
-## Run
+## Run the Quiz App locally
 
-To run the solution, run `npm run dev`.
+### Prerequisites
 
-```bash
-npm run dev
-```
+- A GitHub account
+- [Node.js and Git](https://nodejs.org/)
+
+### Install & Setup
+
+1. Create a repository from this [template](https://github.com/new?template_name=Web-Dev-For-Beginners&template_owner=microsoft) 
+
+1. Clone your new repository, and navigate to the quiz-app
+
+   ```bash
+   git clone https://github.com/your-github-organization/repo-name
+   cd repo-name/quiz-app
+   ```
+
+1. Install the npm packages & dependencies
+
+   ```bash
+   npm install
+   ```
+
+### Build the app
+
+1. To build the solution, run:
+
+   ```bash
+   npm run build
+   ```
+
+### Start the App
 
-## Lint
+1. To run the solution, run:
 
-To ensure the code is linted, run `npm run lint`.
+    ```bash
+    npm run dev
+    ```
+
+### [Optional] Linting
+
+1. To ensure the code is linted, run:
+
+    ```bash
+    npm run lint
+    ```
+
+## Deploy the Quiz-app to Azure 
+
+### Prerequisites
+- An Azure Subscription. _Cost Estimate: FREE_
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.StaticApp)
+
+Once you are signed in on Azure through the link above, select a subscription and resource group then:
+
+- Static Web App details: Provide a name and select a hosting plan
+- GitHub Login: Set your deployment source as GitHub then log in and fill in the required fields on the form:
+    - *Organization* – Choose your organization.
+    - *Repository* – Select the Web Dev for Beginners curriculum repository. 
+    - *Branch* - Select a branch (main) 
+- Build Presets - Azure Static Web Apps uses a detection algorithm to detect the framework used in your application. 
+    - *App location* - ./quiz-app
+    - *Api location* -
+    - *Output location* - dist
+- Deployment: Click 'Review + Create', then 'Create'
+- Post-Deployment: After deployment is complete, click on 'Go to Deployment' then 'View app in browser'.
+
+### Example Workflow File
+
```

Comment:
```suggestion

Once you've deployed your app, a workflow file will have been created in the *.github* directory of your repo.  This workflow file describes how it will run its instructions when a specific event happens like for each a pull request opens or closes to redeploy the app to Azure. 
```

@juliamuiruri4  suggest we add this context above so folks know what a workflow file is, thoughts?